Makefile.PL

```diff
@@ -175,6 +175,9 @@ pure_all :: $(INST_LIBDIR)/LaTeXML/MathGrammar.pm
 
 $(INST_LIBDIR)/LaTeXML/MathGrammar.pm: lib/LaTeXML/MathGrammar
 	$(PERLRUN) -MParse::RecDescent - lib/LaTeXML/MathGrammar LaTeXML::MathGrammar Parse::RecDescent
+	@$(PERLRUN) -e 'exit -e ("$(INST_LIBDIR)/LaTeXML/MathGrammar.pm") ? 0 : 1;' || \
```

Comment:
the leading `@` on this line confused me enough to lose sight of the perlrun for my older question. What does leading `@` do?

Reply:
Prevents the command from being echoed out. I added that to keep users from being confused by a 'fallback' message in the command, even though it is not relevant when not called.

Reply:
I see, thanks for educating me!

Reply:
I remember having known this years ago and then forgetting it, as bash syntax often goes.